client/src/components/DataFiles/DataFilesBreadcrumbs/DataFilesBreadcrumbs.jsx

```diff
@@ -155,52 +199,89 @@ const DataFilesBreadcrumbs = ({
       }
     }, '');
 
+  const lastTwoPaths = paths.slice(-1);
+  const lastTwoPathComps = pathComps.slice(-1);
+  const reversedPath = paths.reverse();
+
   return (
-    <div className={`breadcrumbs ${className}`}>
-      {scheme === 'projects' && (
-        <>
-          <RootProjectsLink
-            api={api}
-            section={section}
-            operation={operation}
-            label="Shared Workspaces"
-          />{' '}
-          {system && `/ `}
-        </>
+    <div className="breadcrumb-container">
+      <div id="path-button-wrapper">
+        <ButtonDropdown
+          isOpen={dropdownOpen}
+          toggle={toggleDropdown}
+          id="go-to-button-dropdown"
+          className="go-to-button-dropdown"
+        >
+          <DropdownToggle tag={Button}>Go to ...</DropdownToggle>
+          <DropdownMenu>
+            {reversedPath.slice(1, reversedPath.length).map((path, index) => {
+              const folderName = path.split('/').pop();
+              return (
+                <>
+                  <Link key={index} to={handleNavigation(path)}>
+                    <DropdownItem>
+                      <i className="icon-folder" />
+                      {folderName.length > 20
+                        ? folderName.substring(0, 20)
+                        : folderName}
+                    </DropdownItem>
+                  </Link>
+                </>
+              );
+            })}
+            <DropdownItem divider />
+            <Link to={handleNavigation(homeDir)}>
+              <DropdownItem>
+                <i className="icon-my-data" />
+                <span className="multiline-menu-item-wrapper">
+                  {systemName || 'Shared Workspaces'}
+                  <small>Root</small>
+                </span>
```

Comment:
Can we vertically align the icon to the root text?
![Screenshot 2023-10-04 at 7 02 25 PM](https://github.com/TACC/Core-Portal/assets/20326896/7d5c68df-15e0-4453-a0e4-40a82055cc6b)

Reply:
I added the complex-menu-item class to the Root dropdown item (same concept as is used in the Sidebar menu) and changed the display to flex and it's aligned correctly again.